diff --git a/core/include/axi_intf.sv b/core/include/axi_intf.sv
index 41d4b16a..20030fcc 100644
--- a/core/include/axi_intf.sv
+++ b/core/include/axi_intf.sv
@@ -16,6 +16,7 @@
 
 
 /// An AXI4 interface.
+`ifndef SPYGLASS
 interface AXI_BUS #(
   parameter AXI_ADDR_WIDTH = -1,
   parameter AXI_DATA_WIDTH = -1,
@@ -259,7 +260,7 @@ interface AXI_LITE #(
   );
 
 endinterface
-
+`endif
 
 /// An AXI routing table.
 ///
